README.md

```diff
@@ -73,13 +73,13 @@ The separation on "logic" and "server" crates is designed to separate functional
 For now, "server" crates contain gRPC and HTTP as the transport layer. 
 It was assumed, that users may want to implement their own APIs, for which the library with functional logic might be used.
 
-Crates that require database connection may also have additional sea-orm defined crates:
+Crates that require database connection may also have additional sea-orm-defined crates:
```

Comment:
why? `sea-orm` is name of library

Reply:
The hyphenation removes ambiguity and ensures the phrase is interpreted correctly as "crates that are defined by sea-orm."

Reply:
at the same time it adds another ambiguity 🫠 
smth like
```
`sea-orm`-defined
```
might be more clear